plans/03-wavy-filter.md

```
@@ -0,0 +1,169 @@
+function WavyFilters({ orbs }: { orbs: WavyOrbConfig[] }) {
+  return (
+    <svg style={{ position: 'absolute', width: 0, height: 0 }} aria-hidden="true">
+      <defs>
+        {orbs.map((orb, i) => (
+          <filter id={`orbkit-wavy-${orb.id}`} key={orb.id}>
+            <feTurbulence
+              type="fractalNoise"
+              baseFrequency={0.01 + (orb.scale ?? 30) * 0.0005}
+              numOctaves={orb.intensity ?? 3}
+              seed={i * 17}
+              result="noise"
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)